src/main/kotlin/no/nav/familie/ef/søknad/integration/UnsecurePdlClient.kt

```diff
@@ -8,27 +8,17 @@ import no.nav.familie.ef.søknad.integration.dto.pdl.PdlBolkResponse
 import no.nav.familie.ef.søknad.integration.dto.pdl.PdlPersonBolkRequest
 import no.nav.familie.ef.søknad.integration.dto.pdl.PdlPersonBolkRequestVariables
 import no.nav.familie.http.client.AbstractRestClient
-import no.nav.familie.http.sts.StsRestClient
 import org.springframework.beans.factory.annotation.Qualifier
 import org.springframework.http.HttpHeaders
 import org.springframework.stereotype.Component
 import org.springframework.web.client.RestOperations
 
 
 @Component
-class PdlStsClient(val pdlConfig: PdlConfig,
-                   @Qualifier("stsRestKlientMedApiKey") restOperations: RestOperations,
-                   private val stsRestClient: StsRestClient)
+class UnsecurePdlClient(val pdlConfig: PdlConfig,
```

Comment:
Usikker på om `Unsecure` er fornuftig prefiks her - trodde først at dette var "åpent - uten autentisering", men det er jo system/applikasjonsbruker involvert her?

Reply:
Poenget med denne er vel at vi må bruke system/applikasjon for autentisering ettersom personen selv ikke har tilgang til data om andre i pdl?

Reply:
Enig med Charlie her. 

Fungerer denne? @Qualifier("clientCredential") testet i preprod? 

Prøvde å se på barnekort hvor barn har annen forelder - mulig det bare er dollytrøbbel/ikke oppdatert ennå, men får ikke opp noe barnekort.

Reply:
Testet med feil bruker (var logget inn som barnet, så ikke hør på meg)

Reply:
@charliemidtlyng Den bruker azure, client_credentials, så den er ju sikker, men den er usikker i den forstand att vi må ha kontroll på bruken av den. 
Jeg vurderte å kalle den noe med ClientCredential, men det sier veldig lite for folk flest.. 
Har du noe forslag? 

@ma10s Fint hvis du sjekker hva som feiler hvis det feiler. Fint om du sjekker på nytt.
 Personen jeg testet med har 1 barn og det kom opp.

Reply:
Skulle i et annet møte, så rakk ikke fullføre testing. Jeg fortsetter test nå :) 

---
ApplikasjonAutentisertPdlClient ??? (Bare tenker høyt)

Reply:
Og etter mye om og (dolly)men fungerte det fint til slutt: Fikk hentet ut barn og barnets andre forelder. 👍 

Bra! (Og beklager maset) 

Eneste jeg ikke får ut riktig i søknaden nå er kallene mot integrasjoner (poststed og land) 

RestClientResponseException ved kall mot uri=https://api-gw-q2.oera.no/familie-integrasjoner/api/kodeverk/poststed
(...)org.springframework.web.client.HttpClientErrorException$BadRequest: 400 Bad Request: [{"status":"400","phrase":"Bad Request","date":"Wed Aug 18 2021 13:15:26 GMT+0200 (CEST)","errors":[{"source":"/familie-integrasjoner/api/kodeverk/poststed","datapower_message":"Access denied. Malforme... (414 bytes)]

Reply:
Oh ser der ja! Sendte ikke med gateway passord, sånn att GW stoppet requesten 
Feil fikset i https://github.com/navikt/familie-ef-soknad-api/pull/504/commits/a5eb09762f13f8812f50c5ce747feddea6aab04a

Fortsatt litt usikker på navnet

Reply:
Supert - 

Navnet ja. Ikke helt sikker selv, men som @charliemidtlyng sier over har vi jo to måter å snakke med pdl på. Kanskje det blir tydeligere hvis vi endrer navn på begge? 

ApplikasjonsautentisertPdlClient, SøkerautentisertPdlClient?

Reply:
Kan `PdlApp2AppClient` funke? STS er jo en teknisk implementasjon i NAV, men App2App er kanskje ikke det ennå?

Reply:
https://github.com/navikt/familie-ef-soknad-api/pull/504/commits/e3da912f7c32261966cda4562cc5ed7e7f31d048